dev/io.openliberty.session.6.1.internal/src/io/openliberty/session61/http/impl/HttpSessionFacadeImpl61.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.session61.http.impl;
+
+import java.util.logging.Level;
+
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.ws.session.AbstractHttpSessionFacade;
+import com.ibm.ws.session.AbstractSessionData;
+import com.ibm.ws.session.utils.LoggingUtil;
+
+import jakarta.servlet.http.HttpSession;
+
+public class HttpSessionFacadeImpl61 extends AbstractHttpSessionFacade {
+    private static final String CLASS_NAME = "HttpSessionFacadeImpl61";
```

Comment:
This applies to all of the new classes, why not get the CLASS_NAME programmatically?